demos-europe/demosplan-core
# Fix twig tool extensions test

Description:
- Make work again TwigToolsExtensionTest

### How to review/test
- Run the test in local, all tests should pass

Delete the checkbox if it doesn't apply/isn't necessary.

- [ x] Tests updated/created
